quipucords/fingerprinter/formatters.py

```diff
@@ -1,5 +1,9 @@
 """Fingerprint formatters."""
 
+import datetime
+from contextlib import suppress
+from venv import logger
```

Comment:
`from venv`? 🤔

Reply:
WTF 🤣 

definitely wrong, lol

thanks for spotting this one